### PR TITLE
Add BT71 entry to F1155/F1255 and fix for BT74

### PR DESCRIPTION
**Pull request by svollebregt**
The Modbusmanager from Nibe is missing this entry for the F1255, while it is included in the list of the F1145/F1245. However, you can add this sensor as AUX input in the software in the heatpump.

When tested locally with HASS with Nibe plugin and a F1255-6 running the latest firmware (R7) this sensor register is correctly read-out and displays the same value as on the heatpump.

### CODE REVIEW

**Comment by svollebregt** [comment]
If one enables the BT74 sensor as AUX input, the coil 47340 in the F-series does not decode correctly. Upon inspection, it seems that value 18 is sent, which I added to the F-series pumps. Tested locally on a F1255-6 with HA.

I do not know if it actually makes sense to change this value through HA, as the heatpump itself only allows BT74 as an option.

File: nibe/data/f1155_f1255.csv [hunk 1]
```diff
@@ -59,6 +59,7 @@ Title;Info;ID;Unit;Size;Factor;Min;Max;Default;Mode
 "EP21-BT3 Return temp S2";"Return temperature for system 2";40129;"�C";s16;10;0;0;0;R;
 "EB100-EP14-BT29 Cpr Oil Temp";"Compressor oil temperature, BT29";40146;"�C";s16;10;0;0;0;R;
 "BT70 HW Comfort Supply Temp.";"Hot water comfort supply temperature, BT70";40147;"�C";s16;10;0;0;0;R;
+"BT71 Ext. Return Temp";"External return temperature, BT71";40152;"�C";s16;10;0;0;0;R;
```
[the remark this answers is not in the document]
Comment (svollebregt) [reply nibe/data/f1155_f1255.csv:62]: No, the CSV from ModbusManager is incorrectly missing this value. From the description I thought only the CSV had to be edited to add new registers. As far as I can tell this value is only missing for the F1155/1255.

[the remark this answers is not in the document]
Comment (elupus) [reply nibe/data/f1155_f1255.csv:62]: Then just add it in the extensions file instead.

[the remark this answers is not in the document]
Comment (svollebregt) [reply nibe/data/f1155_f1255.csv]: Ok, I think I now got it all right.

**Review by elupus — APPROVED** [review]

**Comment by yozik04** [comment]
Thank you @svollebregt and @elupus.

@elupus Do we want to make a release for it or we wait for more changes?

**Comment by elupus** [comment]
Might as well release. There is a new beta upcoming of HA. So i can see if i can get it in.

**Pull request merged**